project/src/main/webapp/script.js

```diff
@@ -167,15 +176,37 @@ async function addCandidateInformation() {
   const socialMedia = dataPackage.socialMediaData;
 
   // Add 3 components of information to the candidate page.
-  addOfficialCandidateInformation(officialCandidateInfo);
   addNewsArticles(newsArticles);
   addSocialMedia(socialMedia);
+  addOfficialCandidateInformation(officialCandidateInfo);
```

Comment:
small nit: on the webpage, official information comes before news articles and social media. Maybe function calls can be made in the same order